core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockHttpGoogleServicesDAO.scala

```diff
@@ -14,27 +14,21 @@ import org.broadinstitute.dsde.rawls.model._
 import scala.collection.mutable
 import scala.concurrent._
 
-class MockBillingHttpGoogleServicesDAO( useServiceAccountForBuckets: Boolean,
+class MockHttpGoogleServicesDAO(
```

Comment:
Personally, I'm suspicious of these sorts of project-wide 'mocks'.
I generally prefer to create a mock where I need it, and specify the methods I want behavior for.
eg: 
`val m = mock[ClassToBeMocked]`

and if I wanted the mock to  return a  specific value on a method, it would just be:
doReturn(the value).when(m.method(argument matchers))

It's lightweight, flexible, and the behavior that's being mocked is clear where it's being used.

Reply:
@blakery and I fought the good fight on trying to create a mock in the test itself instead of instantiating MockBillingHttpGoogleServicesDAO. We failed because of the way HttpGoogleServicesDAO is written, so I am going to stick with this method of mocking for the test.